app/navigation/SettingsNavigation/SettingsNavigation.tsx

```diff
@@ -95,8 +96,10 @@ function Restore({ navigation }: RestoreProps): JSX.Element {
       onFinish: (report) => setImportReport(report),
     });
 
-    await new Promise((res) => setTimeout(res, 1000));
     dispatch(getAllCredentials());
+    dispatch(getAllDidRecords());
+
+    await new Promise((res) => setTimeout(res, 1000));
```

Comment:
Is there a way that we could do this declaratively? I think adding an artificial delay isn't the best solution for two reasons:
* It is not necessarily stable. Different devices may have different results here which could create a bug.
* It imposes an artificial delay into the process that makes the UI slower than it needs to be

If I understand correctly, `setDone(true)` could live in `onFinish` for the `importWallet` call. Then, the dispatch would go through and the UI would react to that update. There may be better patterns, but I figure this is worth looking into.